lib/loader.js

```diff
@@ -16,13 +16,17 @@ module.exports = function(content, map) {
 	var root = query.root;
 	var moduleMode = query.modules || query.module;
 	var camelCaseKeys = query.camelCase || query.camelcase;
+	var sourceMap = query.sourceMap || false;
 	var resolve = createResolver(query.alias);
 
-	if(map !== null && typeof map !== "string") {
+	if(sourceMap && map !== null && typeof map !== "string") {
 		map = JSON.stringify(map);
+	} else {
+		map = null;
```

Comment:
@bebraw https://github.com/webpack-contrib/css-loader/pull/478/commits/caf9873b0e1611d01ec894aeb52bdc7768413af4#diff-bbbe73d87e5520ca3a3e6b634cfae1d9R13 I have already added a test, what else is needed?